backend/StreamServerApp/media_processing.py

```diff
@@ -26,7 +26,7 @@ def convert_subtitles_to_webvtt(input_file, output_file):
 
     """
     cmd = ["ffmpeg", "-n", "-sub_charenc", "UTF-8", "-i", input_file, output_file]
-    if(os.path.isfile(output_file) == False):
+    if(os.path.isfile(output_file) is False):
```

Comment:
When doing conditions on boolean values in Python, we usually would do:
```python
if not os.path.isfile(output_file):
````

Have a look [here](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html) for instance.